packages/fxa-payments-server/src/lib/apiClient.ts

```diff
@@ -73,7 +73,7 @@ async function apiFetch(
     ...options,
     headers: {
       'Content-Type': 'application/json',
-      Authorization: `Bearer ${accessToken}`,
+      Authorization: accessToken ? `Bearer ${accessToken}` : undefined,
```

Comment:
What is the effect of an `Authorization` header of `undefined` here in the auth server? Is this better/different than omitting the header entirely? Looking at [`fxa-auth-server/lib/oauth/client.js`](https://github.com/mozilla/fxa/blob/521338b9f7cc102c348a55e70f02c2e2e640d69a/packages/fxa-auth-server/lib/oauth/client.js#L65-L67), it seems like either would have the same effect as long as we have e.g. a `clientId` in the request body instead?

Reply:
It should be the same as omitting it.